pygoodwe/__init__.py

```diff
@@ -125,6 +125,31 @@ def getCurrentReadings(self, raw=True, retry=1, maxretries=5, delay=30): #pylint
     #                 'eday_kwh': round(eday_kwh, 3)
     #             })
     #     return result
+    
+    def getDayDetailedReadingsExcel(self, date):
+        """ retrieves the detailed daily results of the given date as an Excel sheet
+            processing the Excel sheet is outside the scope of the current module """
+        datestr = datetime.strftime(date, "%Y-%m-%d")
+        payload = {
+            'date' : datestr,
+            'pw_id' : self.system_id,
+            'img_width': 1050,
+            'img_height': 268
```

Comment:
Have you tested changing these settings? If they're settable, I might suggest changing the to be options in the function call, eg:

```python
def getDayDetailedReadingsExcel(self, date, **kwargs):
    <snip>
    "img_width" : kwargs.get("img_width", 1050),
    "img_height" : kwargs.get("img_height", 268),